src/components/Callout.tsx

```diff
@@ -24,48 +25,52 @@ const Callout = ({
   alt,
   titleKey,
   descriptionKey,
+  title,
+  description,
   children,
   className,
-  ...rest
 }: CalloutProps) => {
   const { t } = useTranslation("common")
 
   return (
-    <Flex
-      as="aside"
-      direction="column"
-      bgGradient="linear-gradient(
-    49.21deg,
-    rgba(127, 127, 213, 0.2) 19.87%,
-    rgba(134, 168, 231, 0.2) 58.46%,
-    rgba(145, 234, 228, 0.2) 97.05%
-  )"
-      p={6}
-      m={4}
-      mt={32}
-      mb={{ base: 16, lg: 4 }}
-      borderRadius="base"
-      className={className}
-      {...rest}
+    <aside
+      className={cn(
+        "mb-16 mt-32 flex flex-1 flex-col rounded bg-gradient-to-br from-[rgba(127,127,213,0.2)] via-[rgba(134,168,231,0.2)] to-[rgba(145,234,228,0.2)] p-12 lg:mb-4",
+        className
+      )}
     >
       {image && (
-        <Center maxW="263px" minH="200px" mt={-40} alignSelf="center">
-          <Image src={image} alt={alt || ""} height={200} />
-        </Center>
+        <div className="mt-[-10rem] self-center">
+          <TwImage
+            src={image}
+            alt={alt || ""}
+            className="min-h-[200px] max-w-[263px] object-contain"
+          />
+        </div>
       )}
-      <Flex direction="column" justify="space-between" h="full">
+      <div className="flex h-full flex-col justify-between">
         <div>
           {emoji && <Emoji text={emoji} className="text-5xl" />}
-          <OldHeading as="h3" fontSize="2xl" lineHeight={1.4}>
-            {t(titleKey)}
-          </OldHeading>
-          <Text color="text200" fontSize="xl" lineHeight="140%">
-            {t(descriptionKey)}
-          </Text>
+          {titleKey && (
+            <h3 className="mb-8 mt-10 text-2xl leading-[1.4]">{t(titleKey)}</h3>
+          )}
+          {title && (
+            <h3 className="mb-8 mt-10 text-2xl leading-[1.4]">{title}</h3>
+          )}
+          {descriptionKey && (
+            <p className="mb-6 text-xl leading-[140%] text-body-medium">
+              {t(descriptionKey)}
+            </p>
+          )}
+          {description && (
+            <p className="mb-6 text-xl leading-[140%] text-body-medium">
+              {description}
+            </p>
+          )}
```

Comment:
I'm a little unclear why we're opening up the props to optionally accept `title` and `description` or the keys for these:

<img width="531" alt="image" src="https://github.com/user-attachments/assets/ac96e70d-ec60-4239-8bf4-9193d120e208">

Is the plan to change these components to not have to handle the translation keys? As-is we not longer have any of the props as required.